.circleci/config.yml

```diff
@@ -91,13 +91,10 @@ build_machine_environment: &build_machine_environment
     METEOR_SAVE_TMPDIRS: 1
 
     # Skip these tests on every test run.
-    # For readability, this is a regex wrapped across multiple lines in quotes.
-    SELF_TEST_EXCLUDE: "\
-        ^old cli tests|\
-        ^minifiers can't register non-js|\
-        ^minifiers: apps can't use|\
-        ^compiler plugins - addAssets\
-        "
+    # If needed, for readability this should be a regex wrapped across
+    # multiple lines in quotes.
+    SELF_TEST_EXCLUDE: ""
```

Comment:
Can we get rid of this completely now?

Reply:
I left it there in-case we wanted to add one-off excludes for specific CI configs, like maybe if there were some excludes we needed for Circle but not AppVeyor. Happy to remove them if you don't think this is needed though!